### PR TITLE
feat: AthenaQueryPolicy template

### PR DESCRIPTION
**Pull request by nheijmans**
*Issue #, if available:*
#1136

*Description of changes:*
Added the policy for Athena to the policy_template.json file

*Description of how you validated changes:*
Added the suggested code to the local file and checked with a workgroup if access was granted and queries could be executed. 

*Checklist:*

- [x] Write/update tests
- [x] `make pr` passes
- [ ] Update documentation
- [x] Verify transformed template deploys and application functions as expected
- [ ] Add/update example to `examples/2016-10-31`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: samtranslator/policy_templates_data/policy_templates.json [hunk 1]
```diff
@@ -1736,6 +1736,38 @@
...
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "athena:GetWorkGroup",
+              "athena:GetQueryExecution",
+              "athena:StartQueryExecution",
+              "athena:StopQueryExecution",
+              "athena:GetQueryResults"
```
Comment (ShreyaGangishetty) [inline samtranslator/policy_templates_data/policy_templates.json]: verfied these policies from link: https://docs.aws.amazon.com/athena/latest/APIReference/API_Operations.html

**Comment by ShreyaGangishetty** [comment]
@nheijmans  Thank you for contributing this feature!!
Can you please add unit tests for this policy template [here](https://github.com/awslabs/serverless-application-model/blob/develop/tests/translator/input/all_policy_templates.yaml)?

**Comment by nheijmans** [comment]
> @nheijmans Thank you for contributing this feature!!
> Can you please add unit tests for this policy template [here](https://github.com/awslabs/serverless-application-model/blob/develop/tests/translator/input/all_policy_templates.yaml)?

Added in PR #1158!

**Comment by ShreyaGangishetty** [comment]
@nheijmans I added test cases for this and also resolved the merge conflicts.  This PR looks good to me

File: samtranslator/policy_templates_data/policy_templates.json (hunk 1 above)
Comment (praneetap) [reply samtranslator/policy_templates_data/policy_templates.json]: @nheijmans looking at this AWS docs there are more permissions required for accessing a specified athena workgroup - https://docs.aws.amazon.com/athena/latest/ug/example-policies-workgroup.html
Could you add those as well?

Comment (nheijmans) [reply samtranslator/policy_templates_data/policy_templates.json]: @praneetap Yep I'll add them as well!

File: samtranslator/policy_templates_data/policy_templates.json [hunk 2]
```diff
@@ -1763,6 +1763,59 @@
...
+                "athena:GetNamespaces",
+                "athena:GetTables",
+                "athena:GetTable"
+            ],
+            "Resource": "*"
+        },  
+	{
+            "Effect": "Allow",
+            "Action": [
+    		"athena:StartQueryExecution",
```
Comment (praneetap) [inline samtranslator/policy_templates_data/policy_templates.json]: nit: indentation

**Comment by praneetap** [comment]
@nheijmans The tests need to be updated as well, the build is failing currently. Once you fix that it should be ready to go :)

**Comment by nheijmans** [comment]
> @nheijmans The tests need to be updated as well, the build is failing currently. Once you fix that it should be ready to go :)

How can I do that? Can't find this in GitHub :( 

Sorry for replying late, was on leave...

**Comment by keetonian** [comment]
Rebased and fixed the tests 👍

File: samtranslator/policy_templates_data/policy_templates.json [hunk 3]
```diff
@@ -1795,6 +1795,60 @@
...
+              "athena:GetExecutionEngine",
+              "athena:GetExecutionEngines",
+              "athena:GetNamespace",
+              "athena:GetCatalogs",
+              "athena:GetNamespaces",
+              "athena:GetTables",
+              "athena:GetTable"
+            ],
+            "Resource": "*"
+          },
```
Comment (tuliocasagrande) [inline samtranslator/policy_templates_data/policy_templates.json:1821]: These should be `glue:`

**Pull request merged**